.babelrc

```diff
@@ -1 +1,12 @@
-{ "presets": ["es2015"] }
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "node": "4.0.0"
```

Comment:
It would be great to know the reason behind this change

Reply:
It removes the usage of deprecated `babel-preset-es2015` and switches to `babel-preset-env`, which requires a different config.